src/app/shared/resource-policies/resource-policies.component.ts

```diff
@@ -112,6 +112,7 @@ export class ResourcePoliciesComponent implements OnInit, OnDestroy {
    */
   ngOnInit(): void {
     this.isActive = true;
+    this.requestService.removeByHrefSubstring(this.resourceUUID);
```

Comment:
Is this delete necessary? it's still susceptible to the same problem I described in [my earlier comment](https://github.com/DSpace/dspace-angular/pull/645#issuecomment-624645477): it could inadvertently remove the results that you fetch in this. initResourcePolicyLIst() the line below.

Reply:
it's necessary to clean cache otherwise when you go to the authorization page directly (e.g. `http://localhost:4000/items/3834df7e-ed86-491d-8ad7-beeefa137082/edit/authorizations`), the results are not available after was retrieved on SSR
I've moved after results have been retrieved

Reply:
👍